src/sentry/static/sentry/app/components/group/sidebar.jsx

```diff
@@ -118,7 +119,9 @@ const GroupSidebar = createReactClass({
           <dl key={plugin.slug}>
             <dt>{`${plugin.shortName || plugin.name || plugin.title}: `}</dt>
             <dd>
-              <a href={issue.url}>{issue.label}</a>
+              <a href={issue.url}>
+                {_.isObject(issue.label) ? issue.label.id : issue.label}
```

Comment:
oof i didnt catch this behavior in dev -- is this maybe just because it was returning bad objects here?

https://github.com/getsentry/sentry/pull/7456

Reply:
i think it's because they were objects at all. something this small in the sidebar should block rendering, but we need to implement error boundaries on important components to prevent that